.github/workflows/main.yml

```diff
@@ -341,10 +341,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: gaurav-nelson/github-action-markdown-link-check@v1
+    - uses: gaurav-nelson/github-action-markdown-link-check@1.0.8
```

Comment:
The `v1` tag is a meta version that pulls the latest changes for the v1 release.
This upstream change (in the 1.0.9 release) adds quotes around the "folder-path" provided below which causes `find` to fail.
https://github.com/gaurav-nelson/github-action-markdown-link-check/commit/53c1dda67d2de0a25bbdce0b1a4054cfbb468b32